www/src/scss/obs/styles.scss

```diff
@@ -51,6 +51,15 @@ body {
   position: relative;
   text-align: center;
   width: 100%;
+
+  span.normal-word,
+  span.light-word {
+    word-break: break-word;
+  }
+
+  span.light-word {
+    opacity: 0.5;
```

Comment:
this seems low. for some color choice you can't see it. not sure if its worth optimizing though.